lib/public/components/Filters/common/filters/timeRangeFilter.js

```diff
@@ -0,0 +1,83 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+import { dropdown } from '../../../common/popover/dropdown.js';
+import { h } from '/js/src/index.js';
+import { dateTimeInput } from '../../../common/form/inputs/dateTimeInput.js';
+import { getLocaleDateAndTime } from '../../../../utilities/dateUtils.js';
+
+/**
+ * Format the current timestamp range display
+ *
+ * @param {Partial<Period>} the current period
+ * @return {vnode|string} the resulting component
+ */
+const formatTimeRange = ({ from, to }) => {
+    let parts = [];
+
+    if (from === null && to === null) {
+        return '-';
```

Comment:
I cleaned the code. From and to can not be null, because it's a partial period: they are either a number or `undefined` which makes sense if the period is openened on one/both side.

Reply:
ok